x-pack/plugins/cases/public/components/create/form_context.test.tsx

```diff
@@ -306,6 +306,35 @@ describe('Create case', () => {
       });
     });
 
+    it('should trim fields correctly while submit', async () => {
```

Comment:
Should we test the same for tags and categories?

Reply:
added in [17ed167](https://github.com/elastic/kibana/pull/165867/commits/17ed167a23f02a52077c68988c41079d384b349c)